feedback-form/README.md

```diff
@@ -70,7 +76,67 @@ end
 # .env.example
 # We send all feedback email to this address
 FEEDBACK_EMAIL=test@example.com
+HIPCHAT_ROOM_ID=hip_chat_room_id
+HIPCHAT_AUTH_TOKEN=valid_room_token
+```
+
+```ruby
+# Gemfile
+...
+gem "hipchat"
+...
+```
+
+```ruby
+# app/interactors/submit_feedback.rb
+class SubmitFeedback
+  include Interactor::Organizer
+
+  organize SendFeedbackEmail, SendHipChatNotification
+end
+```
+
+```ruby
+# app/interactors/send_feedback_email.rb
+class SendFeedbackEmail
+  include Interactor
+
+  delegate :feedback, to: :context
+
+  def call
+    ApplicationMailer.feedback(feedback).deliver_now!
+  end
+end
+```
+
+```ruby
+# app/interactors/send_hip_chat_notification.rb
+class SendHipChatNotification
+  include Interactor
+
+  delegate :feedback, to: :context
+
+  def call
```

Comment:
Could you try extract `hip_cd dhat_client` to separate class and define there method for sending message.